From 1af9b599ed2acaa738c1480f383b20d349bb51be Mon Sep 17 00:00:00 2001
From: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date: Tue, 15 Apr 2014 10:56:16 -0400
Subject: [PATCH 597/625] Fix: module instrumentation: update to 3.15 kernel

Remove show_module_flags() define from lttng module.h instrumentation.
It is already defined within the in-kernel module.h, _and_ LTTng does
not use TP_printk.

Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Signed-off-by: Jianchuan Wang <jianchuan.wang@windriver.com>
---
 drivers/staging/lttng2/instrumentation/events/lttng-module/module.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/staging/lttng2/instrumentation/events/lttng-module/module.h b/drivers/staging/lttng2/instrumentation/events/lttng-module/module.h
index c63f866..c2d1ad7 100644
--- a/drivers/staging/lttng2/instrumentation/events/lttng-module/module.h
+++ b/drivers/staging/lttng2/instrumentation/events/lttng-module/module.h
@@ -23,10 +23,6 @@
 #define _TRACE_MODULE_DEF
 struct module;
 
-#define show_module_flags(flags) __print_flags(flags, "",	\
-	{ (1UL << TAINT_PROPRIETARY_MODULE),	"P" },		\
-	{ (1UL << TAINT_FORCED_MODULE),		"F" },		\
-	{ (1UL << TAINT_CRAP),			"C" })
 #endif
 
 TRACE_EVENT(module_load,
-- 
2.0.2

